example/src/simple/component/master.js

```diff
@@ -11,7 +11,7 @@
  *
  */
 
-import utils from './../../../../src/scripts/utils/master';
+import { Component as BasicComponent } from './../../../../src/scripts/component/master';
```

Comment:
Thoughts on adding aliases to avoid paths like this? Or are we trying to stay away completely?